<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUserDialog</class>
 <widget class="QDialog" name="EditUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;class&gt;EditUserDialog&lt;/class&gt;
 &lt;widget class=&quot;QDialog&quot; name=&quot;EditUserDialog&quot;&gt;
  &lt;property name=&quot;geometry&quot;&gt;
   &lt;rect&gt;
    &lt;x&gt;0&lt;/x&gt;
    &lt;y&gt;0&lt;/y&gt;
    &lt;width&gt;556&lt;/width&gt;
    &lt;height&gt;468&lt;/height&gt;
   &lt;/rect&gt;
  &lt;/property&gt;
  &lt;property name=&quot;minimumSize&quot;&gt;
   &lt;size&gt;
    &lt;width&gt;500&lt;/width&gt;
    &lt;height&gt;400&lt;/height&gt;
   &lt;/size&gt;
  &lt;/property&gt;
  &lt;property name=&quot;windowTitle&quot;&gt;
   &lt;string&gt;Редактирование пользователя&lt;/string&gt;
  &lt;/property&gt;
  &lt;property name=&quot;styleSheet&quot;&gt;
   &lt;string notr=&quot;true&quot;&gt;background: #3498db;&lt;/string&gt;
  &lt;/property&gt;
  &lt;widget class=&quot;QGroupBox&quot; name=&quot;groupBox&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;40&lt;/x&gt;
     &lt;y&gt;30&lt;/y&gt;
     &lt;width&gt;450&lt;/width&gt;
     &lt;height&gt;350&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;minimumSize&quot;&gt;
    &lt;size&gt;
     &lt;width&gt;400&lt;/width&gt;
     &lt;height&gt;300&lt;/height&gt;
    &lt;/size&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;background: white&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;title&quot;&gt;
    &lt;string/&gt;
   &lt;/property&gt;
   &lt;layout class=&quot;QVBoxLayout&quot; name=&quot;verticalLayout_2&quot;&gt;
    &lt;item&gt;
     &lt;layout class=&quot;QHBoxLayout&quot; name=&quot;horizontalLayout&quot;&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QLabel&quot; name=&quot;usernameLabel&quot;&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string&gt;Логин&lt;/string&gt;
        &lt;/property&gt;
        &lt;property name=&quot;alignment&quot;&gt;
         &lt;set&gt;Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter&lt;/set&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QLabel&quot; name=&quot;usernameValueLabel&quot;&gt;
        &lt;property name=&quot;styleSheet&quot;&gt;
         &lt;string notr=&quot;true&quot;&gt;font-weight: bold; color: #2c3e50;&lt;/string&gt;
        &lt;/property&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string&gt;admin&lt;/string&gt;
        &lt;/property&gt;
        &lt;property name=&quot;alignment&quot;&gt;
         &lt;set&gt;Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter&lt;/set&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
     &lt;/layout&gt;
    &lt;/item&gt;
    &lt;item&gt;
     &lt;layout class=&quot;QHBoxLayout&quot; name=&quot;horizontalLayout_2&quot;&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QLabel&quot; name=&quot;emailLabel&quot;&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string&gt;Email&lt;/string&gt;
        &lt;/property&gt;
        &lt;property name=&quot;alignment&quot;&gt;
         &lt;set&gt;Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter&lt;/set&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QLineEdit&quot; name=&quot;emailEdit&quot;&gt;
        &lt;property name=&quot;enabled&quot;&gt;
         &lt;bool&gt;true&lt;/bool&gt;
        &lt;/property&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string/&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
     &lt;/layout&gt;
    &lt;/item&gt;
    &lt;item&gt;
     &lt;layout class=&quot;QHBoxLayout&quot; name=&quot;horizontalLayout_3&quot;&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QLabel&quot; name=&quot;roleLabel&quot;&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string&gt;Роль&lt;/string&gt;
        &lt;/property&gt;
        &lt;property name=&quot;alignment&quot;&gt;
         &lt;set&gt;Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter&lt;/set&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QComboBox&quot; name=&quot;roleCombo&quot;&gt;
        &lt;item&gt;
         &lt;property name=&quot;text&quot;&gt;
          &lt;string&gt;Пользователь&lt;/string&gt;
         &lt;/property&gt;
        &lt;/item&gt;
        &lt;item&gt;
         &lt;property name=&quot;text&quot;&gt;
          &lt;string&gt;Оператор&lt;/string&gt;
         &lt;/property&gt;
        &lt;/item&gt;
        &lt;item&gt;
         &lt;property name=&quot;text&quot;&gt;
          &lt;string&gt;Модератор&lt;/string&gt;
         &lt;/property&gt;
        &lt;/item&gt;
        &lt;item&gt;
         &lt;property name=&quot;text&quot;&gt;
          &lt;string&gt;Администратор&lt;/string&gt;
         &lt;/property&gt;
        &lt;/item&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
     &lt;/layout&gt;
    &lt;/item&gt;
    &lt;item&gt;
     &lt;layout class=&quot;QHBoxLayout&quot; name=&quot;horizontalLayout_4&quot;&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QLabel&quot; name=&quot;statusLabel&quot;&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string&gt;Статус&lt;/string&gt;
        &lt;/property&gt;
        &lt;property name=&quot;alignment&quot;&gt;
         &lt;set&gt;Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter&lt;/set&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QComboBox&quot; name=&quot;statusCombo&quot;&gt;
        &lt;item&gt;
         &lt;property name=&quot;text&quot;&gt;
          &lt;string&gt;Активен&lt;/string&gt;
         &lt;/property&gt;
        &lt;/item&gt;
        &lt;item&gt;
         &lt;property name=&quot;text&quot;&gt;
          &lt;string&gt;Заблокирован&lt;/string&gt;
         &lt;/property&gt;
        &lt;/item&gt;
        &lt;item&gt;
         &lt;property name=&quot;text&quot;&gt;
          &lt;string&gt;Приостановлен&lt;/string&gt;
         &lt;/property&gt;
        &lt;/item&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
     &lt;/layout&gt;
    &lt;/item&gt;
    &lt;item&gt;
     &lt;widget class=&quot;QCheckBox&quot; name=&quot;resetPasswordCheckbox&quot;&gt;
      &lt;property name=&quot;text&quot;&gt;
       &lt;string&gt;Сбросить пароль&lt;/string&gt;
      &lt;/property&gt;
     &lt;/widget&gt;
    &lt;/item&gt;
    &lt;item&gt;
     &lt;layout class=&quot;QHBoxLayout&quot; name=&quot;horizontalLayout_5&quot; stretch=&quot;0,1&quot;&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QLabel&quot; name=&quot;passwordLabel&quot;&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string&gt;Новый пароль&lt;/string&gt;
        &lt;/property&gt;
        &lt;property name=&quot;alignment&quot;&gt;
         &lt;set&gt;Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter&lt;/set&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QLineEdit&quot; name=&quot;passwordEdit&quot;&gt;
        &lt;property name=&quot;enabled&quot;&gt;
         &lt;bool&gt;false&lt;/bool&gt;
        &lt;/property&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string/&gt;
        &lt;/property&gt;
        &lt;property name=&quot;echoMode&quot;&gt;
         &lt;enum&gt;QLineEdit::Password&lt;/enum&gt;
        &lt;/property&gt;
        &lt;property name=&quot;placeholderText&quot;&gt;
         &lt;string&gt;Оставьте пустым для сохранения текущего&lt;/string&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
     &lt;/layout&gt;
    &lt;/item&gt;
    &lt;item&gt;
     &lt;widget class=&quot;QCheckBox&quot; name=&quot;showPasswordCheckbox&quot;&gt;
      &lt;property name=&quot;text&quot;&gt;
       &lt;string&gt;Показать пароль&lt;/string&gt;
      &lt;/property&gt;
     &lt;/widget&gt;
    &lt;/item&gt;
    &lt;item&gt;
     &lt;widget class=&quot;QLabel&quot; name=&quot;errorLabel&quot;&gt;
      &lt;property name=&quot;enabled&quot;&gt;
       &lt;bool&gt;true&lt;/bool&gt;
      &lt;/property&gt;
      &lt;property name=&quot;text&quot;&gt;
       &lt;string/&gt;
      &lt;/property&gt;
      &lt;property name=&quot;scaledContents&quot;&gt;
       &lt;bool&gt;false&lt;/bool&gt;
      &lt;/property&gt;
      &lt;property name=&quot;alignment&quot;&gt;
       &lt;set&gt;Qt::AlignCenter&lt;/set&gt;
      &lt;/property&gt;
     &lt;/widget&gt;
    &lt;/item&gt;
    &lt;item&gt;
     &lt;layout class=&quot;QHBoxLayout&quot; name=&quot;horizontalLayout_6&quot;&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QPushButton&quot; name=&quot;cancelButton&quot;&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string&gt;Отмена&lt;/string&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
      &lt;item&gt;
       &lt;widget class=&quot;QPushButton&quot; name=&quot;saveButton&quot;&gt;
        &lt;property name=&quot;text&quot;&gt;
         &lt;string&gt;Сохранить&lt;/string&gt;
        &lt;/property&gt;
       &lt;/widget&gt;
      &lt;/item&gt;
     &lt;/layout&gt;
    &lt;/item&gt;
   &lt;/layout&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
 &lt;connections/&gt;
&lt;/ui&gt;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>450</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Логин</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="roleLabel">
        <property name="text">
         <string>Роль</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="roleCombo">
        <item>
         <property name="text">
          <string>Пользователь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Оператор</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Модератор</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Администратор</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Статус</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="statusCombo">
        <item>
         <property name="text">
          <string>Активен</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Заблокирован</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Приостановлен</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="resetPasswordCheckbox">
      <property name="text">
       <string>Сбросить пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
      <item>
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Новый пароль</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Оставьте пустым для сохранения текущего</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="showPasswordCheckbox">
      <property name="text">
       <string>Показать пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="errorLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
